src/utils/azure.ts

```diff
@@ -82,7 +100,6 @@ export const pickAzureConfig = async (): Promise<IAzureConfig> => {
       }' config...`
     )
   );
-  await getCredentials(config);
```

Comment:
@gunzip 
`getCredentials` (`--overwrite-existing`) is useful when you work with different "endpoints"
without that command you'll get few authentication errors when you switch from an environment to another
ex: you make an op toward `dev` and then you want to make another op toward `prod`